### PR TITLE
{NetAppFiles} Update to 2022-09-01

### PR DESCRIPTION
**Pull request by audunn**
**Related command**
az netappfiles 

**Description**
Update az netappfils to api-version 2022-09-01
**Testing Guide**

**History Notes**

[NetAppFiles] BREAKING CHANGE: `az netappfiles volume create`: Remove optional parameter `--vault-id` as this is not longer needed
[NetAppFiles] BREAKING CHANGE: `az netappfiles vault list`: Remove command `vault list` as this is not longer needed
[NetAppFiles] `az netappfiles account create`: Add optional parameter `--identity-type`
[NetAppFiles] `az netappfiles account ad add`: Add optional parameter `--preferred-servers-for-ldap-client`
[NetAppFiles] `az netappfiles volume create`: Add optional parameter `--is-large-volume`
[NetAppFiles] `az netappfiles volume account create`: Add optional parameter `--identity-type`
[NetAppFiles] `az netappfiles volume quota-rule update`: Add optional parameter `--tags`
[NetAppFiles] `az netappfiles volume`: Add new command `break-file-locks` to break all the file locks on a volume

---
This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/netappfiles/custom.py [hunk 1]
```diff
@@ -55,11 +55,14 @@ def create_account(cmd, client, account_name, resource_group_name, location=None
 
     if key_source is not None and key_source == EncryptionKeySource.MICROSOFT_KEY_VAULT:
         keyVaultProperties = KeyVaultProperties(key_vault_uri=key_vault_uri, key_name=key_name, key_vault_resource_id=key_vault_resource_id)
-        identity = EncryptionIdentity(user_assigned_identity=user_assigned_identity)
-        account_encryption = AccountEncryption(key_source=key_source, key_vault_properties=keyVaultProperties, identity=identity)
+        encryptionIdentity = EncryptionIdentity(user_assigned_identity=user_assigned_identity)
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/netappfiles/custom.py:58]: please use snake case

**Comment by kairu-ms** [comment]
@zhoxing-ms There's a breaking change in this PR.

**Comment by zhoxing-ms** [comment]
> [NetAppFiles] BREAKING CHANGE: az netappfiles volume create: Remove optional parameter --vault-id as this is not longer needed

@kairu-ms Thanks for your kindly reminder. Are you referring to this breaking change? If so, it has already been collected in our breaking change doc (OneNote) by Jeremy

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**